### PR TITLE
[HDWWIZ] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR #5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Review by JoachimHenze — APPROVED** [review]
Approved. No accelerators were touched/changed.

**Comment by GeoB99** [comment]
This PR has merge conflicts. Please fix them.

**Comment by AndreiMiloiu** [comment]
I fixed the merge conflict. It seems that I made two PRs for the same thing. However, in this PR I discovered some typos that needed to be fixed, so now everything seems to be good,

image

File: dll/cpl/hdwwiz/lang/ro-RO.rc [hunk 1]
```diff
@@ -44,49 +44,50 @@ END
...
+    LTEXT "Dacă nu regăsiți categoria pe care o căutați, apăsați pe Afișează toate dispozitivele.", IDC_STATUSTEXT, 21, 9, 275, 18
     LTEXT "Tip&uri obișnuite de dispozitive:", IDC_HWTYPESTEXT, 21, 31, 180, 8
     CONTROL "", IDC_HWTYPESLIST, "SysListView32", WS_BORDER | WS_GROUP | WS_TABSTOP | LVS_REPORT | LVS_NOSORTHEADER | LVS_SINGLESEL | LVS_SHAREIMAGELISTS | LVS_NOCOLUMNHEADER, 21, 42, 275, 92
 END
 
 IDD_PROGRESSPAGE DIALOGEX 0, 0, 317, 143
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Expert de adăugare a dispozitivelor"
+CAPTION "Expert de adăugare de dispozitive"
+
```
Comment (binarymaster) [inline dll/cpl/hdwwiz/lang/ro-RO.rc:90]:
```suggestion
```

**Pull request merged**